README.md

```diff
@@ -169,9 +169,24 @@ make migrate-init
    
 ## Future Node updates
 
-Upon any upcoming Insolar MainNet updates, the current Node version may suspend synching. To resume the synching, you need to update your Node instance.
+Upon any upcoming Insolar MainNet updates, the current Node version may suspend synching. To resume synching, you need to update your Node instance.
```

Comment:
Version does not suspend anything. Node does. Also, remove "you need", just tell them to update it.
Suggestions:
Upon an upcoming Insolar MainNet update, the Node of the current version may suspend synching. To resume synching, update your Node instance.

Reply:
Version does not indeed. But the Node of current version (current Node version as in "current macOS version doesn't support this or that" or "current software version cannot do this or that") does suspend.

https://www.google.com/search?q=%22current+software+version+doesn%27t+support%22&rlz=1C5CHFA_enRU900RU900&oq=%22current+software+version+doesn%27t+support%22&aqs=chrome..69i57j33l7.8065j0j7&sourceid=chrome&ie=UTF-8 732 results is kinda weak, but this is just an exact phrase I googled. I don't get what confuses you here about the phrasing.

Reply:
I adopted your option, but please open the link and maybe try some more phrases like the one I googled. It's standard English, even in tech docs.